### DISCUSSION TITLE
[javascript deep dive] 7장. 연산자

### DISCUSSION THREAD

**Discussion by rladpwl0512** [woowacourse-study/2022-javascript-bank#7] [General]
[javascript deep dive] 7장. 연산자

**Comment by jswith** [comment]
typeof 연산자에 대해 설명해주세요?

**Comment by rladpwl0512** [comment]
연산자가 발생시킬 수 있는 부수효과란 무엇인가요? Javascript에서 부수효과가 있는 연산자의 예시를 알려주세요.

**Comment by rladpwl0512** [comment]
동등비교와 일치비교를 비교해서 설명해주시고, 둘 중 어떤 것을 사용하는 것이 좋을지 근거를 들어 설명해주세요.
